kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1148,6 +1148,15 @@ object KyuubiConf {
       .checkValues(OperationModes.values.map(_.toString))
       .createWithDefault(OperationModes.NONE.toString)
 
+  val OPERATION_PLAN_ONLY_SKIP: ConfigEntry[Seq[String]] =
+    buildConf("operation.plan.only.skip")
+      .doc("Skipped statements or commands in a PARSE, ANALYZE, OPTIMIZE, PHYSICAL, EXECUTION " +
+        "only way without executing the query.")
+      .version("1.5.0")
+      .stringConf
+      .toSequence()
+      .createWithDefault(Nil)
```

Comment:
Let add the Plans that match set/reset/use here to keep the backward compatibility and better UX.